### PR TITLE
Refactored node-forge and lodash imports to reduce size

### PR DESCRIPTION
**Pull request by jfarago**

### CODE REVIEW

File: src/utils/encryption.js [hunk 1]
```diff
@@ -66,22 +73,22 @@ const sideChainGenerate = hash => {
   return sidechain;
 };
 
-const sideChain = hash => sha3_256(forge.util.binary.hex.decode(hash));
+//const sideChain = hash => sha3_256(util.binary.hex.decode(hash));
```
Comment (jfarago) [inline src/utils/encryption.js]: @EdmundMai i was getting an error cause the library reference, but i was not sure where that came from

File: src/datamap.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import iota from "./services/iota";
 import Encryption from "./utils/encryption";
 
 import { FILE } from "./config";
-import forge from "node-forge";
+import util from "node-forge/lib/util";
```
Comment (rfornea) [inline src/datamap.js:5]: Is is possible to just import hexToBytes?

**Review by rfornea — APPROVED** [review]
The only other thing I'd suggest is to see if we can import smaller portions of util rather than all of it.  Other than that everything looks good.

File: src/datamap.js (hunk 2 above)
Comment (jfarago) [reply src/datamap.js:5]: unfortunately, no. They are exporting the entire util object.

**Comment by rfornea** [comment]
Ok

**Pull request merged**